client/src/Pages/AboutUs.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Navbar from '../Components/Navbar'
+
+const AboutUs = () => {
+  return (
+    <div>
+        <Navbar/>
+        AboutUs
+    </div>
+  )
+}
```

Comment:
I tried to adapt this according to the language selected with something like
```suggestion
 const {lng} = useParams();
  if (lng === 'de') {
    return (
      <div>
          <Navbar/>
          Ein website in drei verschiednen Sprachen         
      </div>
    )
  } else if (lng === 'ur') {
    return (
      <div>
          <Navbar/>
          تین مختلف زبانوں میں کئ موضوعات کا احاطہ کرتی ویب سائٹ
      </div>
    )
  } else {
    return (
      <div>
          <Navbar/>
          A multi-language blob portal covering many different themes         
      </div>
    )
  }
```

Is this the correct way or is there another better `React` way

Reply:
No, this isn't the correct way of using multi-langual feature in react app.
Instead of this, this one approach is the optimal one ->
```bash
import { useTranslation } from "react-i18next";

const AboutUs = () => {
  const { t } = useTranslation();

  return (
    <div>
      <Navbar />
      {t("aboutUsText")}
    </div>
  );
};
```
Then, in translation files (`en.json`, `de.json`, `ur.json`):
```bash
{
  "aboutUsText": "A multi-language blog portal covering many different themes"
}
```